src/Dialogs/Weather/RASPDialog.cpp

```diff
@@ -170,19 +212,17 @@ DownloadRASP(const char *url, Path path, ProgressListener &progress)
 void
 RASPSettingsPanel::Download() noexcept
 {
-  unsigned n = 0;
-  for (auto i = rasp_providers; i->url != nullptr; ++i)
-    ++n;
+  unsigned n = items.size();
 
-  assert(n > 0);
+//  assert(n > 0);
```

Comment:
The URL now definded in the repository file... when this can not be loaded properly or does not exists, `n` could be 0... the code does only render this items in a loop. So there will be no need to check the available items